### PR TITLE
Travis CI: Fix build config validation issues

### PR DESCRIPTION
**Pull request by cclauss**
**What kind of change does this PR introduce?**
  - [x] bug fix
  - [ ] feature
  - [ ] docs update
  - [x] tests/coverage improvement
  - [ ] refactoring
  - [ ] other

Screenshot 2020-11-11 at 20 30 56

**What is the related issue number (starting with `#`)**

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior (if this is a feature change)?**

**Other information**:

**Checklist**:

  - [ ] I think the code is well written
  - [ ] I wrote [good commit messages][1]
  - [ ] I have [squashed related commits together][2] after the changes have been approved
  - [ ] Unit tests for the changes exist
  - [ ] Integration tests for the changes exist (if applicable)
  - [ ] I used the same coding conventions as the rest of the project
  - [ ] The new code doesn't generate linter offenses
  - [ ] Documentation reflects the changes
  - [ ] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences

[1]: http://chris.beams.io/posts/git-commit/
[2]: https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -141,12 +139,11 @@ jobs:
     script: skip
     deploy:
       provider: pypi
-      skip_cleanup: true
```
Comment (webknjaz) [inline .travis.yml]: This is important to keep. The warning says that it's `cleanup:` now

**Review by webknjaz — CHANGES_REQUESTED** [review]
Thanks for the PR! There's one problem that needs to be fixed before merging.

File: .travis.yml (hunk 1 above)
Comment (cclauss) [reply .travis.yml]: Let’s revert this change given https://travis-ci.community/t/skip-cleanup-true-is-now-deprecated-how-to-go-for-gradle-publish/6641

File: .travis.yml [hunk 2]
```diff
@@ -146,7 +144,7 @@ jobs:
         tags: true
         all_branches: true
         python: *mainstream_python
-      user: __token__
+      username: __token__
```
Comment (webknjaz) [inline .travis.yml]: https://docs.travis-ci.com/user/deployment-v2/providers/pypi/ says that the default is v1 of dpl that has `user:` and not `username:`

**Pull request closed without merging**